apps/dialtone-documentation/docs/components/presence.md

```diff
@@ -7,6 +7,54 @@ image: assets/images/components/presence.png
 storybook: https://dialtone.dialpad.com/vue/?path=/story/components-presence--default
 figma_url: https://www.figma.com/design/W58r5BkO8qTw3vem9YieJd/DT9-Component-Library--Rebrand-2025-?node-id=9628-58458
 ---
+
+<div class="d-ba d-bc-success d-p32 d-bar16">
+  <h2>Nested Mode Support</h2>
+  <dt-stack gap="500" class="d-mb32">
+    <p>This section is in the page's current mode (inherited from html element).</p>
+    <dt-stack direction="row">
+        <div class="d-w24 d-h24" style="background: var(--dt-color-black-50);" title="black-50"></div>
+        <div class="d-w24 d-h24" style="background: var(--dt-color-black-100);" title="black-100"></div>
+        <div class="d-w24 d-h24" style="background: var(--dt-color-black-200);" title="black-200"></div>
+        <div class="d-w24 d-h24" style="background: var(--dt-color-black-300);" title="black-300"></div>
+        <div class="d-w24 d-h24" style="background: var(--dt-color-black-400);" title="black-400"></div>
+        <div class="d-w24 d-h24" style="background: var(--dt-color-black-500);" title="black-500"></div>
+        <div class="d-w24 d-h24" style="background: var(--dt-color-black-600);" title="black-600"></div>
+        <div class="d-w24 d-h24" style="background: var(--dt-color-black-700);" title="black-700"></div>
+        <div class="d-w24 d-h24" style="background: var(--dt-color-black-800);" title="black-800"></div>
+        <div class="d-w24 d-h24" style="background: var(--dt-color-black-900);" title="black-900"></div>
+        <div class="d-w24 d-h24" style="background: var(--dt-color-black-950);" title="black-950"></div>
+        <div class="d-w24 d-h24" style="background: var(--dt-color-black-1000);" title="black-1000"></div>
+    </dt-stack>
+    <dt-stack data-dt-mode="dark" gap="400" class="d-bgc-primary d-ba d-p16">
+        <div>
+          <h3>Always Dark Section</h3>
+          <p>This section always renders in dark mode, even when the page is in light mode.</p>
+        </div>
+        <dt-stack direction="row">
```

Comment:
Hmm..

Not sure why this is here... should be in mode-island documentation?
I didn't understand what is showing me related to `presence`... if it is intendeed maybe some short description would be useful.


Apart from that, it seems like some text is completely hidden but the white one is still visible in both mode.
I guess it should be readable or maybe not, but should be consistent in both cases.



https://github.com/user-attachments/assets/cf5b8c7b-0a22-4658-8771-a99e4dd040e5

Reply:
Good catch. That was a leftover old explore. Removed.